Automattic/wp-calypso
# Domains: Fix renewing/expiring label for mapped domains when plan is renewing

#### Changes proposed in this Pull Request

This fixes the expires/renewing label to show the plan renewal date for mapped domains, when plan is renewing and not expiring.

<img width="730" alt="Screenshot 2020-07-08 at 10 02 47 PM" src="https://user-images.githubusercontent.com/13062352/86964723-d43fd700-c166-11ea-809d-ca27c8f21302.png">


#### Testing instructions

* Switch to a site which is on a paid plan, with an active payment method, and the plan on auto-renew
* If you do not have a mapped domain, map one
* Before the patch you would see "Expires with your plan on..."
* After the patch, you should see "Renews with your plan on..." when plan's auto-renew is on
* Check if there are any other potential issues 🙏
